### ISSUE TITLE
Atom lint not respecting project .sass-lint file

### ISSUE THREAD

**Issue by igloosi** [AtomLinter/linter-sass-lint#173]
Whole git repo add as a project to Atom.
Sass lint config is in root/config folder
Setting in package is config/.sass-lin.yaml

- Atom version:
- linter-sass-lint version:
- Global sass-lint version (if used):
- Platform:
  -  OSX
- Expected behaviour:
Editor lint should use this config. 
- Actual behaviour / issue description:
Editor lint uses some other, unknown lint file.

Note: gulp-sass-lint CI uses the correct file

**Comment by dennis-best** [comment]
This may not be your problem, but I had some commented text at the top of my .sass-lint.yml. **I removed the comments, and everything worked.**  Oddly, the file worked fine in Sublime (SublimeLinter), but not in Atom.

**Comment by dennis-best** [comment]
This solved my problem. Turns out, when I ran my sass-lint.yml file through a validator (http://www.yamllint.com/) it said this was NOT valid:
```
files:
  include: 'src/sass/**/*.scss'
  ignore:
    - 'src/sass/_shame.scss'
    - 'src/sass/test/_tests.scss'
    - 'src/sass/vendor/**/*.*'
```
This, on the other hand, IS valid:
```
files:
  include:
    - 'src/sass/**/*.scss'
  ignore:
    - 'src/sass/_shame.scss'
    - 'src/sass/test/_tests.scss'
    - 'src/sass/vendor/**/*.*'
```
This fixed my problems in Atom.

**Comment by DanPurdy** [comment]
This should be fixed in v1.8.0 of linter-sass-lint. If you still have issues let me know. I tried with the valid source provided above and all seemed fine.
